src/i18n/ailab.json

```diff
@@ -88,5 +88,6 @@
   "crossTabRelationshipHeader": "Relationship information",
   "crossTabTooMuchData": "The currently-selected data is too large to show in a table.",
   "uniqueOptionsWarningNotice": "Note",
-  "uniqueOptionsWarningMessage": "Categorical columns with more than {valueCount} unique values can not be selected as the label or a feature."
+  "uniqueOptionsWarningMessage": "Categorical columns with more than {valueCount} unique values can not be selected as the label or a feature.",
+  "predictionStatement": "Predict {output} based on {inputs}"
```

Comment:
Does Crowdin highlight these variables? The translator will need to know these are variables and to rearrange them appropriately?

Reply:
Yeah, Crowdin highlights these!
![image](https://user-images.githubusercontent.com/1372238/199847825-f274550c-5b52-4eed-b5d8-52b21bb90787.png)

Reply:
Hopefully the variable names are clear enough that the whole sentence makes sense and they know how to rearrange everything so it reads naturally in their language.